### PR TITLE
yt-dlp 2023.3.4

### PR DESCRIPTION
**Pull request by halilim**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/yt-dlp.rb [hunk 1]
```diff
@@ -3,8 +3,9 @@ class YtDlp < Formula
 
   desc "Fork of youtube-dl with additional features and fixes"
   homepage "https://github.com/yt-dlp/yt-dlp"
-  url "https://files.pythonhosted.org/packages/30/70/578521a22783112ae3dc01e33b23429151edeecb585c7b1ed58feee7fef3/yt-dlp-2023.2.17.tar.gz"
-  sha256 "9af92de5effc193bdb51216d9ebf28874d96180d202fae752b0d9f2a63380f3a"
+  url "https://files.pythonhosted.org/packages/e8/4a/1e01f24fcb49191626e2b17d00e13a093315d03a9da09fccb1c75913e420/yt-dlp-2023.3.4.tar.gz"
+  version "2023.03.04"
```
Comment (chenrui333) [inline Formula/yt-dlp.rb:7]:
```suggestion
```

Comment (chenrui333) [reply Formula/yt-dlp.rb:7]: I think 2023.3.4 is good enough?

Comment (halilim) [reply Formula/yt-dlp.rb]: Not sure, that was what `bump-formula-pr` generated :) Looks like that's the format yt-dlp [is using](https://github.com/yt-dlp/yt-dlp/releases) too. Would it break anything? Though I guess it makes sense not to change that in a version bump anyway.

**Review by nandahkrishna — APPROVED** [review]

File: Formula/yt-dlp.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/yt-dlp.rb]: no, it wont break anything :) (that is good to know it is generated by `bump-formula-pr`)

**Pull request closed without merging**